app/views/posts/show.html.erb

```diff
@@ -9,7 +9,7 @@
         <p class="post"><%=@post.text%></p>
 
     <article class="comments">
-        <%@post.comments.each do |comment|%>
+        <%@post.comments.includes([:author]).each do |comment|%>
```

Comment:
- [ ] Please make sure to correct the `n+1` problem on the right place as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=Make%20sure%20that%20the%20N%2B1%20problem%20is%20solved%20when%20fetching%20all%20posts%20and%20their%20comments%20for%20a%20user.), For example: in the `posts#index` action you should use `@posts = @user.posts.includes(:comments)` instead of `@posts = @user.posts`.

Reply:
@gemmen29, is that the only place, I need to make the changes, `@posts = @user.posts.includes(:comments)`